src/main/java/com/pneumaticraft/commandhandler/CommandHandler.java

```diff
@@ -175,6 +178,68 @@ public void registerCommand(Command command) {
         }
     }
 
+    /**
+     * Parse vanilla selectors. See https://minecraft.gamepedia.com/Commands#Target_selectors about it.
+     *
+     * @param sender  Object that executed the command
+     * @param args    Command arguments
+     * @return Arguments list with selectors parsed to playerName if any.
+     */
+    public List<String> parseSelectors(CommandSender sender, List<String> args) {
+        if (sender == null || args == null) {
+            throw new IllegalArgumentException("sender and args cannot be null!");
+        }
+
+        for (ListIterator<String> argsIterator = args.listIterator(); argsIterator.hasNext(); ) {
+            String arg = argsIterator.next();
+            String parsedArg = parseSingleSelector(sender, arg);
+            if (parsedArg.equals(arg)) {
+                continue;
+            }
+            argsIterator.set(parsedArg);
+        }
```

Comment:
This is a neat approach. I wonder if something like
```java
    return args.stream()
        .map(arg -> parseSingleSelector(sender, arg)
                        .map(parsedArg -> parsedArg.equals(arg) ? arg : parsedArg))
        .collect(Collectors.toList());
```
would work and if it might make it more clear (In this example, `parseSingleSelector` return an `Optional<String>`). What are your thoughts?